src/github/githubRepository.ts

```diff
@@ -203,103 +203,44 @@ export class GitHubRepository implements IGitHubRepository, vscode.Disposable {
 		};
 	}
 
-	async getPullRequests(prType: PRType, page?: number): Promise<PullRequestData | undefined> {
-		return prType === PRType.All ? this.getAllPullRequests(page) : this.getPullRequestsForCategory(prType, page);
-	}
+	async getPullRequestsGraphQL(type: PRType, nextCursor?: string|null):Promise<PullRequestListResponse|undefined> {
+		const { remote, query, octokit } = await this.ensure();
+		const currentUser = octokit && (octokit as any).currentUser;

```

Comment:
just out of curiosity, why `octokit as any`?  Do we not have ts bindings for octokit or something?

Reply:
I have a similar question as I'm pretty sure the bindings exist.

Reply:
Yeah, we do have bindings for octokit, but `currentUser` is something we've added ourselves

https://github.com/Microsoft/vscode-pull-request-github/pull/600#pullrequestreview-167227402

We also added an `octokit.ts` file a while ago that modifies the typings slightly, there was something that should have been nullable that wasn't. `currentUser` can be added there to get rid of our `any` casts. But I think we should ultimately try to get rid of that file by correcting the typings upstream